### ISSUE TITLE
Paste images into Task View

### ISSUE THREAD

**Issue by mpodshivalin** [getting-things-gnome/gtg#477]
What I really like about GTG is the ability to work with a task as a document in a separate window. Everything works really well but one feature I really miss is the ability to paste images inside a task view. My fairly common workflow involves pasting screenshots into task description so I could quickly look at them later.
I understand that this is a difficult feature to get right, but I think it's important.

**Comment by corebots** [comment]
I also feel this missing, also turning them into thumbnails would be great to have. Same for website previews when pasting a link :)

**Comment by zeddo123** [comment]
I see why this could be helpful but wouldn't that make GTG a note taking app which it's not?

**Comment by yash-fn** [comment]
Not necessarily. A picture is worth a thousand words. I personally use a lot of mental maps that I draw. Having these mini maps within the app would better enable me to keep track of tasks for larger projects where the parent task can just be the picture and the sub tasks can detail out element as one would normally.
